### DISCUSSION TITLE
[Feature] Auto-add to album

### DISCUSSION THREAD

**Discussion by Fmstrat** [immich-app/immich#10640] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

**Auto-create/add album for folders on mobile upload**
Use the device name and source folder to auto-create albums and include uploads. For instance:
- `My Pixel 8 - Camera` for the `DCIM/Camera` folder on a Pixel 8

**Auto-create/add album for folders on external libraries**
Basically the same thing, but use the folder name from external libraries if there are subfolders inside it. Would likely need to support updating Album contents when moving files.

Both of these would be great if they were retroactive, such that you could add all previous items once the feature releases, though that may not be possible from mobile uploads.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of a handful of existing feature requests. Please actually use the search next time.

**Comment by Fmstrat** [comment]
@bo0tzz Apologies, but there are 0 open items with "album" and a feature label. Am I missing something?

https://github.com/immich-app/immich/issues?q=is%3Aissue+album+label%3Afeature+is%3Aopen

**Reply by bo0tzz** [reply]
We use discussions for feature requests. https://github.com/immich-app/immich/discussions/categories/feature-request?discussions_q=category%3A%22Feature+Request%22+is%3Aopen+album

**Reply by Fmstrat** [reply]
Thank you. I'm able to see relevant items there.
